images.yaml

```diff
@@ -92,8 +92,25 @@ suggestions:
   - ufoym/deepo:all-py36-cpu
   - ufoym/deepo:all-py27
   - ufoym/deepo:all-py27-cpu
+  - ufoym/deepo:keras-py36-cu90
+  - pytorch/pytorch:1.5.1-cuda10.1-cudnn7-runtime
+  - pytorch/pytorch:1.4-cuda10.1-cudnn7-devel
+  - tensorflow/tensorflow:1.8.0-devel
   - tensorflow/tensorflow:1.13.1-gpu-py3
   - tensorflow/tensorflow:1.13.1-py3
+  - tensorflow/tensorflow:1.15.0-gpu-py3
+  - tensorflow/tensorflow:1.15.2-gpu-py3
+  - tensorflow/tensorflow:2.0.0-gpu-py3
+  - tensorflow/tensorflow:2.0.1-gpu-py3
+  - tensorflow/tensorflow:2.2.0
```

Comment:
Maybe we could actually drop the py2 variants of the newly added libraries too, actually? It's a slightly political decision but I don't really see the reason to hang on to py2 :)